### DISCUSSION TITLE
Replace scenery mode

### DISCUSSION THREAD

**Discussion by Richard-L** [OpenRCT2/OpenRCT2#15765] [Feature requests]
Really just an enhancement suggestion: akin to RCT1 beahviour where fences can be replaced simply by clicking new fence objects on top of existing ones, I believe a more global "replace scenery" mode would aid heavily in building. I would suggest enabling it via ALT+CLICK, since SHIFT and CTRL are already in use.

Holding down ALT would add an icon [like this](https://cdn.onlinewebfonts.com/svg/img_269619.png) to the cursor, and perform an auto-remove scenery before the new object is placed. Ideally it would detect: full tile, 1/4 tile, and fences, depending on the selected object.

**Comment by AaronVanGeffen** [comment]
Since there isn't anything in place for this yet, I would consider this a new feature.

**Comment by Richard-L** [comment]
Interestingly, benches, bins and lamps can be replaced just by building them on the same tile and orientation again. This is also RCT1 behaviour.
